hadoop-connector-examples/README.md

```diff
@@ -38,15 +38,40 @@ Hadoop (verified with Hadoop 2.8.3 on Ubuntu 16.04)
    export PRAVEGA_STREAM=<stream_name> # e.g. myStream
    export CMD=wordcount # so far, can also try wordmean and wordmedian
 
-3. make sure below dirs are empty
+3. make sure input dir is empty
    hadoop fs -rmr ${HADOOP_EXAMPLES_INPUT_DUMMY}
-   hadoop fs -rmr ${HADOOP_EXAMPLES_OUTPUT}
 
-4. generate words into pravega
-   hadoop jar ${HADOOP_EXAMPLES_JAR} randomtextwriter -D mapreduce.randomtextwriter.totalbytes=32000 ${HADOOP_EXAMPLES_INPUT_DUMMY} ${PRAVEGA_URI} ${PRAVEGA_SCOPE} ${PRAVEGA_STREAM}
+4. run hadoop commands
 
-5. run hadoop command
-   hadoop jar ${HADOOP_EXAMPLES_JAR} ${CMD} ${HADOOP_EXAMPLES_INPUT_DUMMY} ${PRAVEGA_URI} ${PRAVEGA_SCOPE} ${PRAVEGA_STREAM} ${HADOOP_EXAMPLES_OUTPUT}
+   4.1 to process all events in the stream
+      // generate new events in pravega
+      hadoop jar ${HADOOP_EXAMPLES_JAR} randomtextwriter -D mapreduce.randomtextwriter.totalbytes=32000 ${HADOOP_EXAMPLES_INPUT_DUMMY} ${PRAVEGA_URI} ${PRAVEGA_SCOPE} ${PRAVEGA_STREAM}
+
+      hadoop fs -rmr ${HADOOP_EXAMPLES_OUTPUT}
+      hadoop jar ${HADOOP_EXAMPLES_JAR} ${CMD} ${HADOOP_EXAMPLES_INPUT_DUMMY} ${PRAVEGA_URI} ${PRAVEGA_SCOPE} ${PRAVEGA_STREAM} ${HADOOP_EXAMPLES_OUTPUT}
+
+   4.2 to only process events in the stream cut
+      // find last line of output of last command, and assign positions to env variable
+      //   e.g.
+      //   End positions of stream cut myScope/myStream: '[[{"scope":"myScope","streamName":"myStream","segmentId":0},48025],[{"scope":"myScope","streamName":"myStream","segmentId":2},41564]]'
```

Comment:
Given that users are not expected to manually construct a `StreamCut` object and the client has already enabled Serialization for streamCuts, the `StreamCut` will have to be represented as a base64 string or a byte array. It will not follow the pattern shown here, right?

Reply:
The pattern only applies to hadoop-connector 3.0 which is bound to pravege-samples. 

Base64 format (or Text Representation of StreamCut after it's ready) will be used to replace the above pattern.

https://github.com/pravega/hadoop-connectors/blob/master/src/main/java/io/pravega/connectors/hadoop/PravegaInputFormat.java#L123

Reply:
Thanks for the clarification.